README.md

```diff
@@ -35,14 +47,29 @@ router.get('/', function (req, res) {
 });
 ```
 
-`index.html`
-```html
-<html>
-	<head></head>
-	<body>
-		<div id="app"><%- content %></div>
-	</body>
-</html>
+### Option: Pass jsx
+
+`index.js`:
+
+```javascript
+var router = require('express')();
+var reactExpressMiddleware = require('react-express-middleware');
+var PageWrapper = require('./page-wrapper.jsx');
```

Comment:
Devil's advocate: I think this is really relevant to the point of this change -- You don't think that would abstract something valuable? It also makes it easier to make the point about how the data passing changes.